dags/openshift_nightlies/dag.py

```diff
@@ -201,6 +202,44 @@ def build(self):
     def _get_openshift_installer(self):
         return rogcp.RoGCPInstaller(self.dag, self.config, self.release)
 
+class HypershiftNightlyDAG(AbstractOpenshiftNightlyDAG):
+    def build(self):
+        mgmt_installer = self._get_openshift_installer()
+        hosted_installer = self._get_hypershift_openshift_installer()
+        install_mgmt_cluster = mgmt_installer.get_install_task()
+        # connect_to_platform = self._get_platform_connector().get_task()
+
+        rosa_post_installation = self._get_rosa_postinstall_setup()._get_rosa_postinstallation()
+
+        # if self.config.cleanup_on_success:
+        #     cleanup_mgmt_cluster = mgmt_installer.get_cleanup_task()
+        #     cleanup_hosted_cluster = hosted_installer.get_cleanup_task()
+        #     install_mgmt_cluster >> rosa_post_installation >> connect_to_platform
+        #     connect_to_platform >> install_hosted_cluster >>  rosa_post_installation >> benchmarks
+        #     benchmarks >> cleanup_hosted_cluster >> cleanup_mgmt_cluster
+        # else:
```

Comment:
I'm assuming this is because it's a WIP but these + L227 need to be uncommented or deleted if they aren't being used.

Reply:
yes, I dont want to dump everything in single PR. I will clean it.